features/provider_rejections.feature

```diff
@@ -0,0 +1,19 @@
+@provider
+Feature: rejections by the provider
+
+  Scenario Outline: A provider can reject applications at various stages
+    Given an application in "<original state>" state
+    When a <actor> <action>
```

Comment:
The "action" definitions below here read oddly in the context of this template string, eg `When a provider reject` — maybe change to `When the X takes action: Y` or equiv? This pattern also occurs in `successful_application_states.feature`

Reply:
Fixed with https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/189/commits/0d252f0cdc321d1cd8568ed204b687e75c4428bc